js/views/audioOnlyScreen.js

```diff
@@ -0,0 +1,83 @@
+/*********************************************************************
+  AUDIO ONLY SCREEN
+*********************************************************************/
+const React = require('React');
+const ControlBar = require('../components/controlBar');
+const ClassNames = require('classnames');
+
+const ScrubberBar = require('../components/scrubberBar');
+
+/**
+ * The screen that is shown while the player state is playing, paused, or ended for the audio only player.
+ * Will display a title with description, a control bar, and a scrubber bar.
+ */
+class AudioOnlyScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      controlBarVisible: true,
+      animate: false
+    };
+  }
+
+  render() {
+    var titleStyle = {
+      color: this.props.skinConfig.startScreen.titleFont.color
+    };
+    var infoPanelClass = ClassNames({
```

Comment:
Is it necessary to use `ClassNames` here considering that none of the classes are dynamic? I think it helps with readability, but just wondering if any of there are supposed to change under certain conditions

Reply:
I'll check this

Reply:
Nope, none of these currently change. Should I leave the ```ClassNames``` usage since it improves readability?

Reply:
@aeng7 I would say yes, although ` 'oo-inactive': false` is probably unnecesary